stream_alert/alert_processor/outputs/demisto.py

```diff
@@ -213,53 +212,82 @@ def custom_fields(self):
     def create_investigation(self):
         return self._create_investigation
 
+    @classmethod
+    def map_severity_string_to_severity_value(cls, severity_string):
+        if not isinstance(severity_string, basestring):
+            return cls.SEVERITY_UNKNOWN
+
+        lc_severity_string = severity_string.lower()
+        if lc_severity_string == 'info' or lc_severity_string == 'informational':
+            return cls.SEVERITY_INFORMATIONAL
+        elif lc_severity_string == 'low':
+            return cls.SEVERITY_LOW
+        elif lc_severity_string == 'med' or lc_severity_string == 'medium':
+            return cls.SEVERITY_MEDIUM
+        elif lc_severity_string == 'high':
+            return cls.SEVERITY_HIGH
+        elif lc_severity_string == 'critical':
+            return cls.SEVERITY_CRITICAL
+
+        return cls.SEVERITY_UNKNOWN
+
 
 class DemistoRequestAssembler(object):
     """Factory class for DemistoCreateIncidentRequest objects"""
 
     @staticmethod
-    def assemble(alert, descriptor):
+    def assemble(alert, alert_publication):
         """
         Args:
-            alert (Alert): Alert instance which triggered a rule
+            alert (Alert): Instance of the alert
+            alert_publication (Dict): Published alert data of the alert that triggered a rule
             descriptor (str): Output descriptor
 
         Returns:
             DemistoCreateIncidentRequest
         """
+        # Default presentation values
```

Comment:
This stuff for demisto looks new (not refactor) right? can you call it out in the PR body